lib/babe/state.go

```diff
@@ -44,12 +45,16 @@ type BlockState interface {
 	GetFinalizedHeader(uint64, uint64) (*types.Header, error)
 	IsDescendantOf(parent, child common.Hash) (bool, error)
 	NumberIsFinalised(num *big.Int) (bool, error)
+	HandleRuntimeChanges(newState *rtstorage.TrieState, in runtime.Instance, bHash common.Hash) error
```

Comment:
I don't think this is used in this package

Reply:
Done